apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/GosysOppgaveModal.tsx

```diff
@@ -33,6 +33,13 @@ const ButtonRow = styled.div`
   gap: 1rem;
   justify-content: flex-end;
 `
+const gosysUrl = await hentGosysUrlApi().then((res) => {
```

Comment:
Bygget feiler: 
> Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 2 overrides)

Spm er da om vi ønsker å gå for "esnext" og forvente nyere versjoner, eller om `React Context`-tilnærmingen kanskje likevel heller bør anvendes.

Reply:
Er det ein farbar veg å leggje dette i ein useEffect?

Reply:
Hva jeg ser når jeg tester det, så kan man ikke ha staten utenfor komponenten og da blir det altså trigget for hver komponent. Ønsker ikke å hente urlen for hver eneste gosys-oppgave.

Reply:
Den feilmeldingen kommer av at du bruker `await` utenfor en `async function`. Hvis du legger den inn i det så fungerer det.
Tenker at du kan legge den bak `useApiCall` som kalles via en `useEffect` med args `[]` slik at det kjøres ved mount av komponenten feks.

Reply:
Jeg tror at vi egentlig ønsker å laste inn denne URLen høyere opp i treet slik at du slipper problemet med å unngå og laste den inn for hver eneste GOSYS oppgave. 
Man kan feks legge den inn i REDUX staten slik at du kan hente den ut derifra og kun laste den inn når man laster applikasjonen. Feks i `ToggleNyOppgaveliste` komponenten.